src/state/slices/opportunitiesSlice/resolvers/foxfarming.ts

```diff
@@ -132,14 +246,73 @@ export const foxFarmingLpUserDataResolver = async ({
   opportunityType: OpportunityDefiType
   accountId: AccountId
   reduxApi: ReduxApi
-}): Promise<{ data: string }> => {
+}): Promise<void> => {
```

Comment:
Didn't make sense to return `{data}` since we don't consume it and just made discriminating the staking resolver a pain we could avoid by simply returning void